Jenkinsfile

```diff
@@ -71,11 +71,11 @@ def nodeYaml(version) {
       |      imagePullPolicy: Always
       |      resources:
       |        requests:
-      |          memory: "2Gi"
-      |          cpu: "650m"
+      |          memory: "2490Mi"
```

Comment:
Is it worth adding a comment to ref the internal document? something like `// see 'kubePodTemplate sizing' internal pipeline document for more details`

Reply:
Perhaps we should just move these templates alongside all the others, see d77f8b0